allspice/utils/bom_generation.py

```diff
@@ -455,6 +466,10 @@ def _component_attributes(component: dict) -> ComponentAttributes:
         attributes["_unique_id"] = component["unique_id"]
     except KeyError:
         pass
+    try:
```

Comment:
Should probably update the doc comment for the function that says "This also adds two properties of the component".

Reply:
I meant at the top of this function. But as long as it's on your radar, that's fine.